getting-started/what-you-need.md

```diff
@@ -1,102 +1,116 @@
+* The [ZIP archive]({%slug installation/zip%}) package:
 
-* The [ZIP archive]({%slug installation/zip%}) package - You can download it from your [Telerik account](https://www.telerik.com/account/) and then use it without an Internet connection. It does not provide information about new versions and does not require installation. It also provides an offline version of the [demos](https://demos.telerik.com/blazor-ui/).
+   * You can download it once from your [Telerik account](https://www.telerik.com/account/) and then use it without an Internet connection.
+   * It does not provide information about new versions and does not require installation. 
+   * It provides an offline version of the [demos](https://demos.telerik.com/blazor-ui/).
 
-* The standalone `.nupkg` files - They are the bare minimum that is required. To use them, follow the instructions for using the [ZIP archive]({%slug installation/zip%}), but download the `.nupkg` files instead.
+* The standalone `.nupkg` files:
 
-@[template](/_contentTemplates/common/get-started.md#nuget-update-note)
+   * They are the bare minimum that is required. 
+   * To use them, follow the instructions for using the [ZIP archive]({%slug installation/zip%}), but download the `.nupkg` files instead.
 
+## Adding the Client Assets
 
-## Client Assets
+For the Telerik UI for Blazor components to look and behave as expected in the browser, add the [Telerik stylesheet](#telerik-stylesheet) and the [JS Interop file](#telerik-js-interop-file) to your main index file. Depending on the Blazor hosting model and framework version, your project's main index file will differ:
 
-To have the Telerik Blazor components look and behave as expected in the browser, you need some assets. 
-@[template](/_contentTemplates/common/js-interop-file.md#app-paths)
+   * For Client-Side Blazor apps, use the `wwwroot/index.html` index file.
+   * For Server-Side Blazor apps, use one of the following index files:
+      * `~/Pages/_Host.cshtml` for .NET 3.x and .NET 5
+      * `~/Pages/_Layout.cshtml` for .NET 6
 
+### Telerik Stylesheet
 
-#### The Telerik stylesheet - For more information, refer to the [Themes]({%slug general-information/themes%}) article. Here is a short example:
+The stylesheet allows you to use one of the built-in [Themes]({%slug general-information/themes%}), for example, the Default theme:
 
 @[template](/_contentTemplates/common/js-interop-file.md#theme-static-asset-snippet)
 
 
-#### The Telerik JS Interop file - Provides features that cannot be implemented with native Blazor.
+### Telerik JS Interop File
+
+The JS Interop file provides features that cannot be implemented with native Blazor.
 
 @[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
 
 
-### Static Assets
+### Using Static Assets
```

Comment:
The section order and overall algorithm is a bit odd. First, we demonstrate usage of static NuGet assets for the CSS/JS files. Then we start explaining the possible options and the reader may feel thrown back and forth between options. Can we streamline everything a bit?

Reply:
I rearranged the section to streamline the content.